packages/waagent/Jenkinsfile

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2024 VyOS maintainers and contributors
+//
+// This program is free software; you can redistribute it and/or modify
+// in order to easy exprort images built to "external" world
+// it under the terms of the GNU General Public License version 2 or later as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+@NonCPS
+
+// Using a version specifier library, use 'current' branch. The underscore (_)
+// is not a typo! You need this underscore if the line immediately after the
+// @Library annotation is not an import statement!
+@Library('vyos-build@current')_
+
+def package_name = 'waagent'
+def pkgList = [
+    ['name': "${package_name}",
+     'scmCommit': 'debian/2.9.1.1-2',
+     'scmUrl': 'https://salsa.debian.org/cloud-team/waagent.git',
+     'buildCmd': 'sudo mk-build-deps --install --tool "apt-get --yes --no-install-recommends"; sudo apt-get install --yes --no-install-recommends dpkg-source-gitarchive; ../build.py'],
```

Comment:
As there are no external patches to be applied, do we really need: `../build.py`?

Why not simply build the DEB using: `dpkg-buildpackage -uc -us -tc -b`?

Reply:
It is just a common approach to build all new packages from me - it is easier to keep one standard script and logic for everything than sanitize and add features on demand. Consider this as a part of my build system. :)

But if you want to keep everything at a bare minimum, I have no objections to removing it.

Reply:
Who will make sure to remove this custom waagent once this issue have been resolved upstream (in future updates of this packages at Debian)?

Reply:
Packe is removed by the maintainers from the VyOS repo if required